padle/padle.py

```diff
@@ -0,0 +1,537 @@
+                                    subs=[], all_scores=[], save_daily_scores=[])
+
+        self._daily_padle_loop = bot.loop.create_task(self.generate_padle())
+
+    async def get_dbcog(self) -> "DBCog":
+        dbcog = self.bot.get_cog("DBCog")
+        if dbcog is None:
+            raise ValueError("DBCog cog is not loaded")
+        await dbcog.wait_until_ready()
+        return dbcog
+
+    @commands.group()
+    async def padle(self, ctx):
+        """Commands pertaining to PADle"""
+
+    @padle.command()
+    async def help(self, ctx):
+        """Instructions for PADle"""
+        prefix = (await self.bot.get_prefix(ctx.message))[0]
+        await ctx.send("- PADle is similar to Wordle, except for PAD cards.\n"
+                       "- You have infinite tries to guess the hidden PAD card (chosen from a list "
+                       "of more well-known monsters). Everyone is trying to guess the same PAD card. "
+                       "With each guess, you are given feedback as to how similar the two cards are, "
+                       "including comparing the awakenings, rarity, typings, attributes, and monster "
+                       "point sell value.\n- A new PADle is available every day. "
+                       "Use `{}padle start` to begin!".format(prefix))
+
+    @padle.command(aliases=["sub"])
+    async def subscribe(self, ctx, sub_arg=True):
+        """Subscribe to daily notifications of new PADles"""
+        prefix = (await self.bot.get_prefix(ctx.message))[0]
+        subbed_users = await self.config.subs()
+        if not sub_arg:
+            if ctx.author.id in subbed_users:
+                return await self.unsubscribe(ctx)
+            return await send_cancellation_message(ctx, "You are not subscribed.")
+        if ctx.author.id not in subbed_users:
+            async with self.config.subs() as subs:
+                subs.append(ctx.author.id)
+            return await send_confirmation_message(ctx, "You will now recieve notifications of new PADles. "
+                                                   "You can unsubscribe with `{}padle subscribe 0`.".format(prefix))
+
+        confirmation = await get_user_confirmation(ctx, "You are already subscribed. Did you mean to unsubscribe?")
+        if confirmation:
+            await self.unsubscribe(ctx)
+        elif confirmation is None:
+            await send_cancellation_message(ctx, "Confirmation timeout")
+        else:
+            await ctx.send("No changes were made, you will still recieve notifications of new PADles.")
+
+    async def unsubscribe(self, ctx):
+        async with self.config.subs() as subs:
+            subs.remove(ctx.author.id)
+        return await send_confirmation_message(ctx, "You will no longer recieve notifications of new PADles.")
+
+    @padle.command()
+    async def start(self, ctx):
+        """Start a game of PADle"""
+        if(await self.config.user(ctx.author).done()):
```

Comment:
PEP-8 says to always put a space after an `if`.  These parentheses are redundant anyway.